src/server/game/Battlegrounds/ArenaTeam.h

```diff
@@ -120,7 +122,7 @@ struct ArenaTeamStats
     uint32 Rank;
 };
 
-#define MAX_ARENA_SLOT 3                                    // 0..2 slots
+#define MAX_ARENA_SLOT 4                                    // 0..2 slots
```

Comment:
I guess the comment `0..2 slots` should be either removed or updated

Reply:
Oh, i didn't see this, i will update.